proto/auction/v1/genesis.proto

```diff
@@ -18,4 +18,10 @@ message GenesisState {
 
 message Params {
   uint64 price_max_block_age = 1;
+  uint64 minimum_bid_in_usomm = 2;
+  uint64 blocks_to_not_prune = 3;
```

Comment:
Using a negative here is a tad confusing, we may want to consider something like `auction_max_block_age` or something along those lines.